packages/vaex-ml/vaex/ml/__init__.py

```diff
@@ -103,6 +103,20 @@ def minmax_scaler(self, features=None, feature_range=[0, 1], prefix='minmax_scal
         return minmax_scaler
 
 
+    def groupby_transformer(self, by, agg, rprefix='', rsuffix=''):
```

Comment:
we dont autogen these?

Reply:
Not in this API. We do in the sklearn-like API, where we use the traitlets..